### DISCUSSION TITLE
Problem with verifying my student account

### DISCUSSION THREAD

**Discussion by GutowskiO** [community/community#58942] [GitHub Education]
### Hi everyone,

I am a student at high school and after 4 rejections I start to feel like giving up. My school doesn't give either me or my teachers e-mails so I can't use them. First time as proof I sent my ID, and I was rejected because I didn't set up github profile. I did it and then I got rejected again for this reason: If your school does not provide academic email then you must enable use of your device camera to capture your image. I made a selfie with my ID and then I got rejected again for the same reason. Then I sent my report card that I have just received and I got rejected for the reason. Why is this happening? What do I do?

**Comment by ElBe-Plaq** [comment]
You can try sending both in one picture. I do not know if this will work though, because I got rejected 11 times now :(

**Reply by GutowskiO** [reply]
Yeahh I sent a photo of me with my ID in my hand and still nothing. All of this seem pretty werid. hoepfully we will be accepted one day. What is the reason for your rejections? The same as mine?

**Reply by ElBe-Plaq** [reply]
Mostly, because I have to change my bio and name, even though I changed them.
